src/variables.scss

```diff
@@ -31,6 +31,7 @@ $swal2-error: #f27474 !default;
 $swal2-warning: #f8bb86 !default;
 $swal2-info: #3fc3ee !default;
 $swal2-question: #87adbd !default;
+$swal2-icon-font-family: 'Open Sans', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', Helvetica, Arial, sans-serif !default;
```

Comment:
So far we didn't get any single complain about the face that the icon's `font-family` is inherited from an app which mean people are happy with the current behavior.

I'm happy with the new SASS variable, but do you agree with setting it to`inherit`?

If devs aren't happy with `inherit` (which didn't happen so far), they always have the ability to set a custom icon class or redefine the SASS variable.

Reply:
> So far we didn't get any single complain

Agree. I think my comment goes more towards creating a _branding_ for SweetAlert2. It's nice to think that SweetAlert2 has its own characteristics that makes it unique and recognizable (e.g. the pop-up animation, the success icon). Looking at that from this perspective, one could argue that we may want to make sure that some of the graphical elements of the modal should have specific _default styling_ (that the user can always override). And the icons are one of this distinctive elements. Hence my comment. 

Does that make any sense?

Reply:
Yes, it does. Having the default styling for icons would be a breaking change though.

Reply:
We can add the variable with `inherit` here, and then change `inherit` to default font for the next major.

Reply:
Sounds great to me and agree that is a breaking change! Thanks @limonte